test/e2e/tests/interpreters/default-interpreters.test.ts

```diff
@@ -72,4 +72,29 @@ test.describe('Default Interpreters', {
 		// hidden interpreter (Conda)
 		await app.workbench.console.waitForConsoleContents('3.12.9', { expectedCount: 1 });
 	});
+
+	test('R - Add a default interpreter', async function ({ app, userSettings, runCommand }) {
+
+		await app.workbench.console.waitForInterpretersToFinishLoading();
+
+		// close qa-example-content
+		await runCommand('workbench.action.closeFolder');
+
+		await expect(async () => {
+			await app.workbench.console.waitForInterpretersToFinishLoading();
+
+			// hidden interpreter
+			await userSettings.set([['positron.r.interpreters.default', '"/home/runner/scratch/R-4.4.1/bin/R"']], false);
+		}).toPass({ timeout: 45000 });
+
+		await app.workbench.console.waitForReadyAndStarted('>', 30000);
+
+		await app.workbench.console.barClearButton.click();
+
+		await app.workbench.console.pasteCodeToConsole('cat(R.version.string, "\n")');
+		await app.workbench.console.sendEnterKey();
+
+		// hidden interpreter
+		await app.workbench.console.waitForConsoleContents('4.4.1', { expectedCount: 1 });
```

Comment:
I might suggest using `R.home()` to get a path in addition to or instead of the version? As it stands, it feels like this could pass for the wrong reasons, i.e. default discovery happens to cough up R 4.4.1.

Reply:
Oh, ok.  I can add that once I get the basic test passing.  I was just relying on the version because its the only 4.4.1 on the runner and its hidden from discovery (in ~/scratch).

Reply:
It turns out that this validation was actually validating an unintended side effect and not the primary goal of seeing the default interpreter show up first in the interpreter dropdown.